cfme/tests/configure/test_workers.py

```diff
@@ -140,3 +140,74 @@ def _ui_check():
     mem_threshold_real = get_memory_threshold_in_advanced_settings(appliance, worker)
     MESSAGE = "memory threshold have changed incorrectly in advanced settings"
     assert mem_threshold_real == f"{change_val}.gigabytes", MESSAGE
+
+
+@test_requirements.settings
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1348625])
+def test_verify_purging_of_old_records(request, appliance):
+    """
+    Verify that tables are being purged regularly.
+
+    Bugzilla:
+        1348625
+
+    Polarion:
+        assignee: tpapaioa
+        casecomponent: Appliance
+        initialEstimate: 1/6h
+        startsin: 5.8
+    """
+    old_settings = appliance.advanced_settings
+
+    @request.addfinalizer
+    def _restore_advanced_settings():
+        appliance.update_advanced_settings(old_settings)
+        appliance.evmserverd.restart()
+
+    purge_settings = {
+        'container_entities_purge_interval': '5.minutes',
+        'binary_blob_purge_interval': '5.minutes',
+        'compliance_purge_interval': '5.minutes',
+        'drift_state_purge_interval': '5.minutes',
+        'event_streams_purge_interval': '5.minutes',
+        'notifications_purge_interval': '5.minutes',
+        'performance_realtime_purging_interval': '5.minutes',
+        'performance_rollup_purging_interval': '5.minutes',
+        'policy_events_purge_interval': '5.minutes',
+        'report_result_purge_interval': '5.minutes',
+        'task_purge_interval': '5.minutes',
+        'vim_performance_states_purge_interval': '5.minutes'
+    }
+    new_settings = {'workers': {'worker_base': {'schedule_worker': purge_settings}}}
+    appliance.update_advanced_settings(new_settings)
+    appliance.evmserverd.restart()
+
+    obj_types = (
+        'Binary blobs',
+        'Compliances',
+        'Container groups',
+        'Container images',
+        'Container nodes',
+        'Container projects',
+        'Container quota items',
+        'Container quotas',
+        'Containers',
+        'daily metrics',
+        'Drift states',
+        'Event streams',
+        'hourly metrics',
+        'Miq report results',
+        'Miq tasks',
+        'Notifications',
+        'orphans in Vim performance states',
+        'Policy events'
+    )
+
+    def _log_purge_check():
```

Comment:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/log_validator.py#L21

Check out the log validator to check for this message.